templates/nextflow-aurora-mysql.yaml

```diff
@@ -53,6 +53,7 @@ Mappings:
       version: 5.7.mysql_aurora.2.07.1
       engine: aurora-mysql
       family: aurora-mysql5.7
+      logtypes: ["audit", "error", "general", "slowquery"]
```

Comment:
@tthyer: I included all possible log types since I assume we want to be exhaustive, but maybe this isn't the right approach. Notably, do we care about `slowquery`? If so, we might want to customize the `long_query_time` MySQL parameter. I'm not a database expert, so I defer to others for what would be a sensible value.

Reply:
Yes, normally you care very much about slowquery. Let's include it. What's the default for long_query_time?

Reply:
From what I can tell, it's 10 seconds. More details on [JIRA](https://sagebionetworks.jira.com/browse/WORKFLOWS-17?focusedCommentId=132137).